lib/rubocop/cop/layout/space_in_lambda_literal.rb

```diff
@@ -25,19 +25,20 @@ class SpaceInLambdaLiteral < Cop
 
         ARROW = '->'.freeze
         MSG_REQUIRE_SPACE = 'Use a space between `->` and opening ' \
```

Comment:
I guess we can even omit "opening", as with `(` it's clear that it's an opening parenthesis.

Reply:
Sure! I fixed it. Thanks for your review.